modules/system/database/migrations/2022_08_06_000026_Db_System_Add_App_Birthday_Date.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Winter\Storm\Database\Updates\Migration;
+use Carbon\Carbon;
+use System\Models\Parameter;
+use System\Models\PluginVersion;
+
+class DbSystemAddAppBirthdayDate extends Migration
+{
+    public function up()
+    {
+        $appBirthday = Parameter::get('system::app.birthday');
+
+        if ($appBirthday) {
+            return $appBirthday;
+        }
+
+        $appBirthdayFromPlugins = PluginVersion::orderBy('created_at')->first()?->created_at;
+        $appBirthday = $appBirthdayFromPlugins ? Carbon::parse($appBirthdayFromPlugins)->timestamp : Carbon::now()->timestamp;
+
+        Parameter::set('system::app.birthday', $appBirthday);
+
+        return $appBirthday;
+    }
```

Comment:
```suggestion
    public function up()
    {
        $appBirthday = Parameter::get('system::app.birthday');
        if ($appBirthday) {
            // Return early if the birthday has already been set
            return;
        }

        $firstPluginCreated = PluginVersion::orderBy('created_at')
            ->first()
            ?->created_at;

        $appBirthday = $firstPluginCreated ?: Carbon::now();

        Parameter::set('system::app.birthday', $appBirthday);
    }
```

Reply:
created_at is a Carbon instance when set and Carbon will handle being turned into a string for storage and the System\Helpers\DateTime will handle turning it back into a Carbon instance when rendering.